tests/bcl-test/SyncTestResources.targets

```diff
@@ -0,0 +1,9 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<PropertyGroup>
+	    <CreateAppBundleDependsOn>$(CreateAppBundleDependsOn);CopyFiles</CreateAppBundleDependsOn>
+	</PropertyGroup>
+
+	<Target Name="CopyFiles">
+	    <Exec Command="rsync -r $(AppBundleDir)/Contents/Resources/Test/ $(AppBundleDir)/Contents/MonoBundle/Test/" />
```

Comment:
worth adding an (XML) comment in the file. I'm not likely to remember this next year and the PR comment will be lost (at least hard to track).